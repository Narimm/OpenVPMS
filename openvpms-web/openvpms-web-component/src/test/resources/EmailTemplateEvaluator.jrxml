<?xml version="1.0" encoding="UTF-8"?>
<!-- Created with Jaspersoft Studio version 6.1.0.final using JasperReports Library version 6.1.0  -->
<!-- 2016-03-07T09:43:24 -->
<jasperReport xmlns="http://jasperreports.sourceforge.net/jasperreports" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jasperreports.sourceforge.net/jasperreports http://jasperreports.sourceforge.net/xsd/jasperreport.xsd" name="blank" pageWidth="595" pageHeight="842" columnWidth="555" leftMargin="20" rightMargin="20" topMargin="20" bottomMargin="20" uuid="b558af14-cc21-4477-9d6f-a9599e690cfd">
	<parameter name="OpenVPMS." class="java.lang.String"/>
	<field name="name" class="java.lang.String"/>
	<background>
		<band splitType="Stretch"/>
	</background>
	<title>
		<band height="1" splitType="Stretch"/>
	</title>
	<pageHeader>
		<band splitType="Stretch"/>
	</pageHeader>
	<columnHeader>
		<band height="30" splitType="Stretch">
			<staticText>
				<reportElement x="0" y="0" width="100" height="30" uuid="ad683029-63b1-4c4f-8eb4-d4d22f2504bf"/>
				<text><![CDATA[name]]></text>
			</staticText>
		</band>
	</columnHeader>
	<detail>
		<band height="30" splitType="Stretch">
			<textField>
				<reportElement x="0" y="0" width="100" height="30" uuid="ee926fbc-bd60-45e7-a7d0-7a7819559bb0"/>
				<textFieldExpression><![CDATA[$F{name}]]></textFieldExpression>
			</textField>
		</band>
	</detail>
	<columnFooter>
		<band splitType="Stretch"/>
	</columnFooter>
	<pageFooter>
		<band splitType="Stretch"/>
	</pageFooter>
	<summary>
		<band splitType="Stretch"/>
	</summary>
</jasperReport>
