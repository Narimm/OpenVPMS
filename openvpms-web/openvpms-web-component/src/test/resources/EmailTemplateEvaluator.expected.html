<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
    <title></title>
</head>
<body><a name="JR_PAGE_ANCHOR_0_1"></a>
<table class="jrPage" cellpadding="0" cellspacing="0" border="0" style="empty-cells: show; width: 595px; border-collapse: collapse; background-color: white;">
<tr valign="top" style="height:0">
<td style="width:20px"></td>
<td style="width:100px"></td>
<td style="width:475px"></td>
</tr>
<tr valign="top" style="height:21px">
<td colspan="3">
</td>
</tr>
<tr valign="top" style="height:30px">
<td>
</td>
<td style="text-indent: 0px; text-align: left;">
<span style="font-family: 'DejaVu Sans', Arial, Helvetica, sans-serif; color: #000000; font-size: 10px; line-height: 1.2578125;">name</span></td>
<td>
</td>
</tr>
<tr valign="top" style="height:30px">
<td>
</td>
<td style="text-indent: 0px; text-align: left;">
<span style="font-family: 'DejaVu Sans', Arial, Helvetica, sans-serif; color: #000000; font-size: 10px; line-height: 1.2578125;">Smith,Joe</span></td>
<td>
</td>
</tr>
<tr valign="top" style="height:761px">
<td colspan="3">
</td>
</tr>
</table>

<![if IE]>
<script>
var links = document.querySelectorAll('link.jrWebFont');
setTimeout(function(){ if (links) { for (var i = 0; i < links.length; i++) { links.item(i).href = links.item(i).href; } } }, 0);
</script>
<![endif]>
</body>
</html>